app/models/user.rb

```diff
@@ -16,6 +16,26 @@ class User < ActiveRecord::Base
   has_many :magics
   has_many :universes
 
+  # as_json would try to print the password digest, which requires authentication
+  def as_json(options={})
+    excludes = [:password_digest, :old_password]
```

Comment:
Would we also want to exclude `encrypted_password` and `reset_password_token`? And maybe `email`, so we're not exposing user emails to anyone? What do you think?